artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -266,10 +266,12 @@
    @Option(name = "--no-fsync", description = "Disable usage of fdatasync (channel.force(false) from java nio) on the journal")
    private boolean noJournalSync;
 
+   @Option(name = "--device-block-size", description = "The block size by the device, default at 4096.")
```

Comment:
should this be journal-device-block-size.

e.g. what occurs in future where want to tune the others, and they maybe using other disks with other block sizes, might be best to namespace this now, so in future doesnt cause any issue

Reply:
ok.. I will send another PR with journal-*

Reply:
everything else is already with journal on front.. I only missed it here.